homeassistant/components/haveibeenpwned/sensor.py

```diff
@@ -25,17 +25,19 @@
 MIN_TIME_BETWEEN_FORCED_UPDATES = timedelta(seconds=5)
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
 
-URL = 'https://haveibeenpwned.com/api/v2/breachedaccount/'
+URL = 'https://haveibeenpwned.com/api/v3/breachedaccount/'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_EMAIL): vol.All(cv.ensure_list, [cv.string]),
+    vol.Required(CONF_API_KEY): vol.All(cv.string),
```

Comment:
Remove `vol.All`. That's only needed if more than one validator should be combined.